README.md

```diff
@@ -18,7 +18,27 @@ NYC-DB builds a postgresql database containing the following datasets:
 - 2017 Marshal Evictions
 - ECB / Oath Hearings
 
-NYC-DB is a python3 command line program that downloads and loads datasets into postgres.
+NYC-DB includes a python3 command line program that downloads the data and loads datasets into postgres.
+
+While loading data into postgres, it also transforms the raw data in various ways: by cleaning up bad data, formatting it more consistently, creating indexes on frequently-used columns, and more.
+
+## Database structure
+
+The datasets in NYC-DB come from disparate sources, but almost all of them can be
+linked together through a single identifier: the **Borough Block Lot (BBL) number**.
+From the [NYC Open Data Glossary][glossary]:
+
+> **Borough Block Lot (BBL) number:** A combination of three numeric codes -- a 1-digit borough number, a block number (up to 5 digits) and a lot number (up to 4 digits) -- designated and modified by the Department of Finance (DOF). BBLs are used by various city agencies to identify real estate for taxes, zoning, construction, and other purposes.
+
+The BBL isn't ideal in many ways: sometimes, for example, multiple buildings can exist on the same lot. However, it's an identifier that's been around for a very long time, and one that almost every existing dataset already contains, in one form or another. One of the key goals behind NYC-DB is to ensure that as many datasets as possible have a consistently-formatted ten-digit `bbl` column, so that joins can be made between tables as easily and reliably as possible.
```

Comment:
I would get rid of the part about BBL isn't ideal, because I'm not sure what that means. I agree that it is important to understand the difference between a tax lot (identifier: BBL) and a building (identifier: BIN) and that people commonly forget that tax lots can have multiple buildings.